.editorconfig

```diff
@@ -0,0 +1,15 @@
+# for the sake of sanity
```

Comment:
What IDE/Editor uses `.editorconfig`?

Reply:
Every editor that provides a plugin for it or supports it natively, like (Neo)Vim, VSCode, or IntelliJ IDEs.